### ISSUE TITLE
List is not rendering correctly

### ISSUE THREAD

**Issue by amy-jung** [makerdao/community-portal#167]
## Bug description
List is not rendering correctly on "Community-projects/index"

## Image
<img width="1054" alt="Screen Shot 2020-07-14 at 7 01 55 PM" src="https://user-images.githubusercontent.com/26228406/87454621-857dbb80-c604-11ea-86fe-b54042592a85.png">

## Code

```
<List>

<Box>

### 1. Write your proposal using our template.

Use this Google Doc template to help structure a proposal so that information is clear and milestones are defined.

<Link to="https://docs.google.com/document/d/1lF8mNuomrguIS3lnvqTomS4NdON40nuyKRbLS6ZkMoA/edit">Open proposal template</Link>

</Box>

<Box>

### 2. Post your proposal in our chat.

Post your proposal in our <Link to="https://chat.makerdao.com/channel/community-development">Community-Development chat</Link>. An Advisor will reach out to help you through next steps.

An Approving Group meets weekly to review proposals and will be in touch for regular updates. For more detail, read about the [community project process](./community-projects-process.md).

</Box>

</List>
```

**Comment by MaximumCrash** [comment]
I've come to the conclusion that the code provided above doesn't meet the spec defined for lists currently. 

The reason it's rendering this way is because markdown is recognizing both boxes as 1 child. This is due to the number of new lines between the boxes. 

If you write it like this instead (notice the extra new line between boxes):

```
<List>

<Box>

### 1. Write your proposal using our template.

Use this Google Doc template to help structure a proposal so that information is clear and milestones are defined.

<Link to="https://docs.google.com/document/d/1lF8mNuomrguIS3lnvqTomS4NdON40nuyKRbLS6ZkMoA/edit">Open proposal template</Link>

</Box>


<Box>

### 2. Post your proposal in our chat.

Post your proposal in our <Link to="https://chat.makerdao.com/channel/community-development">Community-Development chat</Link>. An Advisor will reach out to help you through next steps.

An Approving Group meets weekly to review proposals and will be in touch for regular updates. For more detail, read about the [community project process](./community-projects-process.md).

</Box>

</List>
```

you will get:

![Screen Shot 2020-07-15 at 8 26 58 AM](https://user-images.githubusercontent.com/9157341/87544811-0d9ba980-c675-11ea-878f-986b029a5f5f.png)


This writing style is unfavorable and I hope that (if I have time to simplify lists I will do so before v1 launch). I've mentioned this here: #125. Because Lists are in flux currently I've provided a spec you can reference here: #133

**Comment by MaximumCrash** [comment]
@amy-jung  For the sake of making things look good you can copy and paste this to get what you need for now:

```
<List>

- ### 1. Write your proposal using our template.
   - <Box>

    Use this Google Doc template to help structure a proposal so that information is clear and milestones are defined. 

    <Link to="https://docs.google.com/document/d/1lF8mNuomrguIS3lnvqTomS4NdON40nuyKRbLS6ZkMoA/edit">Open proposal template</Link>

    </Box>


- ### 2. Post your proposal in our chat.
	- <Box>

		Use this Google Doc template to help structure a proposal so that information is clear and milestones are defined. 

		<Link to="https://docs.google.com/document/d/1lF8mNuomrguIS3lnvqTomS4NdON40nuyKRbLS6ZkMoA/edit">Open proposal template</Link>
	
	  </Box>

</List>
```

**Comment by amy-jung** [comment]
ahh sounds good. Will keep an eye out. Good news is we decided to get rid of this content-wise and use an accordion (but we will be using list somewhere else).

**Comment by MaximumCrash** [comment]
@all-contributors please add @amy-jung for doc

**Comment by MaximumCrash** [comment]
@amy-jung Your way of writing lists has become the status quo. 

This has been closed by #199
